testgame.pde

```diff
@@ -76,7 +76,73 @@ void setup() {
     ball_x = 1;
 };
 
-void draw() {
+

```

Comment:
straightforward edits that look good to go.

Doing a code review to build the good practice.